src/coreclr/tools/Common/Compiler/DependencyAnalysis/Target_LoongArch64/LoongArch64Emitter.cs

```diff
@@ -144,5 +144,11 @@ public void EmitJE(Register regSrc, ISymbolNode symbol)
 
             EmitJMP(symbol);
         }
+
+        // dbar
+        public void EmitDMB()
```

Comment:
```suggestion
        public void EmitDBAR()
```
The name of the method should match the name of the instruction.

Reply:
OK, thanks.